client/jetpack-connect/auth-form-header.jsx

```diff
@@ -20,6 +20,7 @@ import { authQueryPropTypes } from './utils';
 import { decodeEntities } from 'lib/formatting';
 import { getAuthorizationData } from 'state/jetpack-connect/selectors';
 import { getCurrentUser } from 'state/current-user/selectors';
+import getPartnerSlugFromQuery from 'state/selectors/get-partner-slug-from-query';
```

Comment:
Nit: would you like to alphabetize the order of these imports?

Reply:
Isn't this already alphabetized? The beginning of the last few imports is this:

`getA`
`getC`
`getP`

Reply:
That being said, alphabetizing those is more of a nitpick on our side, don't feel obliged to do it and feel free to ship without addressing that if you prefer. 😉